react/NestedSelect/NestedSelect.md

```diff
@@ -95,6 +98,30 @@ const InteractiveExample = () => {
   const handleClickLeftRadio = () => {
     setLeftRadio(!leftRadio)
   }
+  
+  const handleClickWithSearch = (e) => {
+    if (e.target.checked) {
+    const searchOpts = {
+        placeholderSearch: 'Placeholder Search',
+        noDataLabel: 'No Data Found',
+        onSearch: (value) => {
+          return options.children.filter(o => o.description && o.description.toLowerCase().includes(value.toLowerCase()))
```

Comment:
We should also search on the title

Reply:
Why ? this is for example

Reply:
Because exemples should be as close as possible to our production components